### ISSUE TITLE
UI fork messaging is misleading.

### ISSUE THREAD

**Issue by petong** [AugurProject/augur#356]
UI says all that is needed is 5.5M REP migrated and the Forking market card says over 5.5M REP but the finalize button doesn't show. 

Reason is the market card shows the 5% bonus which doesn't count in the 5.5M REP calculations

![screen shot 2018-10-21 at 11 06 32 am](https://user-images.githubusercontent.com/11491365/49775091-cc2e6580-fcab-11e8-9f01-fa67f859ceec.png)

![screen shot 2018-10-21 at 11 06 48 am](https://user-images.githubusercontent.com/11491365/49775095-d0f31980-fcab-11e8-989f-15a8324bd388.png)
...

@tomhaile So is the expected behavior here that the amount migrated should not include the 5% bonus?
 -adrake33
...

It shows how much REP is on the universe, not how much has been migrated. User can subtract 5% to get actual amount of REP migrated.

Issue as I see it; if we are telling the user 5.5M REP needs to be migrated to end the fork early but the number they are seeing doesn't tell them amount of REP actually migrated, it seems hard to follow. Also, I'm not sure if this is ever going to happen.

Not sure what the best solution is. 

We could subtract 5% on the migration form and the forking market card and leave the actual REP on universe on the universe selection page. I think we need some discussion.
 -bthaile
...

@serena @mattbullock What would your preference be in how this is displayed to the user?
 -adrake33
...

I agree with Tom. The market card should show the amount of REP actually migrated (along with the 5.5 REP message). So let's subtract out 5% of those values before they're displayed on the market card.
 -sharkcrayon
...

Moving this to the backlog for now since the fork was canceled.
 -adrake33

**Comment by phoebemirman** [comment]
related to https://github.com/AugurProject/augur/issues/429

**Comment by bthaile** [comment]
There is a new forking design, that will address verbiage.
